src/content/ui-components/install/astro.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Astro
+meta:
+  title: Astro | Tiptap UI Components
+  description: Install and configure Tiptap UI Components in your Astro project.
+  category: UI Components
+---
+
+Use the command below to scaffold a new Astro project with TypeScript.
+
+```bash
+npx create-astro@latest astro-app  --template with-tailwindcss --install --add react --git
+```
+
+## Edit tsconfig.json file
+
+Add the following code to the `tsconfig.json` file to resolve paths:
+
+```json
+{
+  "compilerOptions": {
+    // ...
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
+    }
+  }
+}
+```
+
+## Add Tiptap components
+
+Install Tiptap UI components using the CLI. For example, to add the `HeadingButton` component:
+
+```bash
+npx @tiptap/cli add heading-button
+```
+
+The command above will install the `HeadingButton` component and its dependencies. You can then import and use it in your **Tiptap** project:
+
+```tsx
+import React from 'react'
+import { HeadingButton } from '@/components/ui/heading-button'
+
+export default function EditorPage() {
+  // Tiptap setup...
+
+  return (
+    <div className="editor-page">
+      <div className="toolbar">
+        <HeadingButton level={1}>Heading 1</HeadingButton>
+        <HeadingButton level={2}>Heading 2</HeadingButton>
+        <HeadingButton level={3}>Heading 3</HeadingButton>
+      </div>
+      {/* Your editor component */}
+    </div>
+  )
+}
+```
+
+## Add Styles
+
+Before proceeding, ensure that your project includes a CSS reset. If you're using Tailwind CSS, you can skip this step since it comes with a built-in reset.
```

Comment:
```suggestion
Before proceeding, ensure that your project includes a CSS reset. If you're using Tailwind CSS, you can skip this step since it comes with a built-in preset.
```